packages/react-components/src/components/Dialpad/Dialpad.styles.ts

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IStyle, ITextFieldStyles, Theme } from '@fluentui/react';
+
+/**
+ * @private
+ */
+export const containerStyles = (theme: Theme): IStyle => {
+  return {
+    width: '18.75rem',
+    padding: '1.25rem',
+    margin: '1.5rem',
+    textAlign: 'center',
+    background: `${theme.palette.white}`,
+    borderRadius: '0.75rem'
+  };
+};
+
+/**
+ * @private
+ */
+export const rowStyles = (theme: Theme): IStyle => {
+  return {
+    textAlign: 'center',
+    display: 'flex',
+    flexDirection: 'row',
+    flexWrap: 'nowrap',
+    justifyContent: 'space-around',
+    alignItems: 'baseline',
+    background: `${theme.palette.white}`
+  };
+};
+
+/**
+ * @private
+ */
+export const digitStyles = (theme: Theme): IStyle => {
+  return {
+    padding: '0.625rem 1.875rem',
+    fontSize: '1.25rem',
+    fontWeight: 600,
+    color: `${theme.palette.neutralPrimary}`,
+    border: 'none',
+    background: `${theme.palette.white}`,
```

Comment:
Setting a hard background here means hover, focus etc. won't have any background

Reply:
Looks like Fluent's standard button already has no background https://developer.microsoft.com/en-us/fluentui#/controls/web/button, can we use this:
![image](https://user-images.githubusercontent.com/2684369/167484086-34936243-ffcb-4b76-83be-3be00fe20945.png)

Reply:
updated